src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -234,6 +234,10 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                             torrent.name = regex.Replace(torrent.name, "$1" + ReplaceMulti + "$2");
                         }
 
+                        var publishDate = DateTimeUtil.UnixTimestampToDateTime(torrent.added);
+                        var guid = new Uri(TorrentDescriptionUrl.Replace("{id}", torrent.id.ToString()));
```

Comment:
I think you can remove all the code related to guid and use the link instead, is unique.

Reply:
I'll add a TODO to this one as well, as I don't want to change something until I know why it is the way it is.